yarn.lock

```diff
@@ -7333,7 +7172,7 @@ react-test-renderer@^15.3.2:
     fbjs "^0.8.9"
     object-assign "^4.1.0"
 
-react@^15.1.0:
+react@^15.1.0, react@^15.4.1:
```

Comment:
@chrislloyd are we sure that this won't load 2 versions of React when people use Gestalt?

Reply:
Good catch! However I don’t think this affects package installing as only the package.json version ranges are respected there for libs. What this does mean however is that either the docs or tests are using a different version of React which is problematic.